libs/designsystem/src/lib/components/segmented-control/segmented-control.component.spec.ts

```diff
@@ -70,61 +70,11 @@ describe('SegmentedControlComponent', () => {
     expect(component.value).toBe(items[1]);
   });
 
-  describe("in 'default' mode", () => {
-    it("should have a 'default' mode when created", () => {
-      expect(component.mode).toBe(SegmentedControlMode.default);
-    });
-
-    it('should have a segment button per item', () => {
-      expect(spectator.queryHostAll('ion-segment-button').length).toBe(items.length);
-    });
-
-    it('should not have any segmented chips', () => {
-      expect(spectator.queryHostAll('kirby-chip').length).toBe(0);
-    });
-
-    it('should call onSegmentSelect when ionChange event fires', async () => {
-      expect(component.value).toBe(items[1]);
-      const ionSegmentElement = spectator.queryHost<HTMLIonSegmentElement>('ion-segment');
-      await TestHelper.whenReady(ionSegmentElement);
-      spyOn(component, 'onSegmentSelect');
-      const changeEvent = new CustomEvent('ionChange', { detail: { value: items[0].id } });
-      ionSegmentElement.dispatchEvent(changeEvent);
-      expect(component.onSegmentSelect).toHaveBeenCalledWith(items[0].id);
-    });
-
-    it('should set value to event.detail.value when ionChange event fires', async () => {
-      expect(component.value).toBe(items[1]);
-      const ionSegmentElement = spectator.queryHost<HTMLIonSegmentElement>('ion-segment');
-      await TestHelper.whenReady(ionSegmentElement);
-      const changeEvent = new CustomEvent('ionChange', { detail: { value: items[2].id } });
-      ionSegmentElement.dispatchEvent(changeEvent);
-      expect(component.value).toBe(items[2]);
-    });
-
-    it('should have touch area with minimum size equal to fat finger size', () => {
-      const touchArea = window.getComputedStyle(
-        spectator.element.querySelector('ion-segment-button'),
-        '::after'
-      );
-
-      expect(parseInt(touchArea.height)).toBeGreaterThanOrEqual(parseInt(fatFingerSize()));
-      expect(parseInt(touchArea.width)).toBeGreaterThanOrEqual(parseInt(fatFingerSize()));
-    });
-
-    describe('when updating items', () => {
-      it('should not emit segmentSelect event', async () => {
-        const ionSegmentElement = spectator.queryHost<HTMLIonSegmentElement>('ion-segment');
-        await TestHelper.whenReady(ionSegmentElement);
-
-        const clonedItems = JSON.parse(JSON.stringify(items));
-        spectator.setHostInput({ items: clonedItems });
-        expect(onSegmentSelectSpy).not.toHaveBeenCalled();
-      });
-    });
-  });
-
-  const testModes = [SegmentedControlMode.chip, SegmentedControlMode.compactChip];
+  const testModes = [
```

Comment:
This whole files diff makes the changes look more advanced than they actually are. 

What actually happens is that default mode is added to the testModes array, and then the whole `in 'default' mode` that looks deleted here is modified slightly to fit all modes. 

Then the other part of the test suite testing the chip mode is deleted.